docs/user/guides/subprojects.rst

```diff
@@ -0,0 +1,32 @@
+How To Add a Subproject
+=======================
+
+This guide shows you how to add a subproject to your main project.
+
+.. seealso::
+
+   :doc:`/subprojects`
+      Read more about what the subproject feature can do and how it works.
+
+Adding a subproject
+-------------------
+
+In the admin dashboard for your project, select :guilabel:`Subprojects` from the menu.
+From this page you can add a subproject by typing in the project slug.
+
+Subproject aliases
+~~~~~~~~~~~~~~~~~~
+
+You can choose an alias for the subproject when it is created.
+This allows you to override the URL that is used to access it,
+giving more control over how you want to structure your projects.
+
+You can set your subproject's project name and slug however you want,
+but we suggest prefixing it with the name of the main project.
```

Comment:
```suggestion
You can set your subproject's project name and :term:`slug` however you want,
but we suggest prefixing it with the name of the main project.
```

We say URL above, and slug here. I'm not sure what this sentence is saying though? 

I think it's generally best to avoid using `slug`, since it's definitely jargon.